src/runner.py

```
@@ -115,50 +131,56 @@ def create(
         image: str,
         resources: VirtualMachineResources,
         binary: Path,
-        reconcile_interval: int,
         registration_token: str,
     ):
+        """Create the runner instance on LXD and register it on GitHub.
+
+        Args:
+            image: Name of the image to launch the LXD instance with.
+            resources: Resource setting for the LXD instance.
+            binary: Path to the runner binary.
+            registration_token: Token to register the runner on GitHub.
+
+        Raises:
+            RunnerCreateError: _description_
+        """
         logger.info("Creating runner: %s", self.name)
 
         self.instance = self._create_instance(image, resources)
 
         try:
-            self._start_instance(reconcile_interval)
+            self._start_instance()
+            # Wait some initial time for the instance to boot up
+            time.sleep(60)
+            self._wait_boot_up()
             self._install_binary(binary)
             self._configure_runner()
             self._register_runner(registration_token, labels=[self.app_name, image])
             self._start_runner()
         except Exception as err:
             self.instance.stop(wait=True)
-            try:
-                self.instance.delete(wait=True)
-            except Exception:  # nosec B110
-                # this is just a fall-back.
+
+            with suppress(Exception):
```

Can we make this more explicit in which exception is being suppressed?